src/inventory_client/inventory_client.go

```diff
@@ -222,6 +223,16 @@ func (c *inventoryClient) GetEnabledHostsNamesHosts(ctx context.Context, log log
 	return c.GetHosts(ctx, log, []string{models.HostStatusDisabled})
 }
 
+func (c *inventoryClient) FilterHostsByStatus(hosts map[string]HostData, status string) map[string]HostData {
```

Comment:
lets move it to assisted-installer/src/common/common.go

Reply:
at least i think so. I think that functions in inventory client is meant to use inventory client no?